test/unit/domain_test.rb

```diff
@@ -167,8 +167,17 @@ def create_a_host
     assert Domain.first.nameservers.empty?
   end
 
-  test "should query remote nameservers" do
-    assert Domain.first.nameservers.empty?
+  test "should query remote nameservers from domain SOA" do
```

Comment:
This seemingly unrelated change is because of the test_helper fix, which stops setting changes leaking between tests (previously only in integration tests).  This test relied on the behaviour of the previous test.